keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/ResourceController.kt

```diff
@@ -94,17 +94,6 @@ class ResourceController(
     resourcePauser.resumeResource(id)
   }
 
-  @Deprecated("Individual resource creation is deprecated. Please use a delivery config.")
```

Comment:
Removal of the resource create/delete endpoints since it's for real no longer supported